### PR TITLE
[SW-370] Refactor: Notifications statuses as constants and tabs styling

### PR DESCRIPTION
**Pull request by peronczyk**

### CODE REVIEW

**Review by konradodo — APPROVED** [review]

**Review by pstachel — APPROVED** [review]

File: src/popup/components/Modals/TransferReceive.vue [hunk 1]
```diff
@@ -2,7 +2,7 @@
   <Modal
     class="transfer-receive-modal"
     has-close-button
-    full-screen
+    from-bottom
```
Comment (CedrikNikita) [inline src/popup/components/Modals/TransferReceive.vue:5]: @pstachel Can you eloborate on that?
https://github.com/aeternity/superhero-wallet/commit/9c8dd0bffc0bab3f3e945aed7d67218060f00087#diff-d6de021fcbbb90f085a88939079166a4ecc2411dee451a285f308e1fa3e8af1cR5

File: src/popup/components/Header.vue [hunk 2]
```diff
@@ -37,10 +37,12 @@
       v-if="isLoggedIn"
       class="right"
     >
-      <NotifyBell />
+      <NotifyBell
+        v-if="showNotifications"
+      />
```
Comment (CedrikNikita) [inline src/popup/components/Header.vue:42]: The style of the project is such that one attribute should not be on the next line.
Just like in that example. 
https://eslint.vuejs.org/rules/max-attributes-per-line.html

Comment (peronczyk) [reply src/popup/components/Header.vue:42]: Yeah, I know. But there are 2 elements after this one with the v-if directives in separate lines so I wanted to maintain same flow here. But sure, I'll change this.

File: src/popup/components/Modals/TransferReceive.vue (hunk 1 above)
Comment (peronczyk) [reply src/popup/components/Modals/TransferReceive.vue:5]: This was already discussed on priv. Paweł made a temporary change on his branch and forgot to bring back the correct prop. I'm fixing this here.

File: src/popup/components/Header.vue [hunk 3]
```diff
@@ -108,15 +110,19 @@ export default {
       return this.$route.meta.modalHeader;
     },
     showBack() {
-      return (this.$route.meta.backButton !== undefined ? this.$route.meta.backButton : true)
-        && this.title;
+      return !!this.$route.meta.backButton;
```
[the remark this answers is not in the document]
Comment (peronczyk) [reply src/popup/components/Header.vue:113]: Yeah, that's definitely my mistake. I'll bring back the previous code.

**Comment by CedrikNikita** [comment]
In general I am against making random changes in a very specifily named commits 
`refactor(notifications): statuses as constants and tabs styling`.

**Review by CedrikNikita — APPROVED** [review]
Code changes look reasonable.

**Pull request merged**